docs/application/web/guides/text-input/input-method.md

```diff
@@ -130,7 +133,9 @@ Tizen has the following additional configuration elements:
   ```
 
 > **Note**  
-> 2-letter primary codes are reserved for [ISO639] language abbreviations. 2-letter codes include `fr` (French), `de` (German), `it` (Italian), `nl` (Dutch), `el` (Greek), `es` (Spanish), `pt` (Portuguese), `ar` (Arabic), `he` (Hebrew), `ru` (Russian), `zh` (Chinese), `ja` (Japanese), `hi` (Hindi), `ur` (Urdu), and `sa` (Sanskrit).Any 2-letter subcode is understood to be a [ISO3166] country code. For more information, see [http://www.w3.org/TR/html401/struct/dirlang.html](http://www.w3.org/TR/html401/struct/dirlang.html).
+> 2-letter primary codes are reserved for [ISO639] language abbreviations. 2-letter codes include `fr` (French), `de` (German), `it` (Italian), `nl` (Dutch), `el` (Greek), `es` (Spanish), `pt` (Portuguese), `ar` (Arabic), `he` (Hebrew), `ru` (Russian), `zh` (Chinese), `ja` (Japanese), `hi` (Hindi), `ur` (Urdu), and `sa` (Sanskrit).  
+> 
+> Any 2-letter subcode is understood to be a [ISO3166] country code. For more information, see [http://www.w3.org/TR/html401/struct/dirlang.html](http://www.w3.org/TR/html401/struct/dirlang.html).
 
 ## Hardware Key Events
 
```

Comment:
line 207: Style was not applied in the Code Block. (such as 'abc' or 'abc')
line 212: Style was not applied in the Code Block. (displaying 'def')
line 213: Style was not applied in the Code Block. (or 'def' string)

Reply:
As we can't apply font style of sub-line or background in code block, I removed the sample string of sub-line and background.